python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -30,6 +32,21 @@ def test_tidy_config(self):
         self.assertEqual('invalid config table [wrong]', errors.next()[2])
         self.assertNoMoreErrors(errors)
 
+    def test_directory_checks(self):
+        mock_config = {
+            "check_ext": {
+                "./python/tidy/servo_tidy_tests/dir_check/webidl_plus/": ['webidl', 'test'],
+                "./python/tidy/servo_tidy_tests/dir_check/only_webidl/": ['webidl'],
+                "./python/tidy/servo_tidy_tests/dir_check/empty/": ['webidl']
+            }
+        }
+        with mock.patch.dict('servo_tidy.tidy.config', mock_config):
```

Comment:
And, we won't be needing the `mock` library to patch the config along the way. We could just pass a custom `dict` to the function.